api/rpc/client/client.go

```diff
@@ -22,6 +23,14 @@ var (
 	// staticClient is used for generating the OpenRPC spec.
 	staticClient Client
 	Modules      = moduleMap(&staticClient)
+
+	RequestURL string
+)
+
+const (
+	authEnvKey = "CELESTIA_NODE_AUTH_TOKEN"
```

Comment:
this might be making go lint blow up, we have label this for the linter to ignore elsewhere, give it a try and see if the go-lint check stops acting foolish

Reply:
Locally everything is working for me lol

Reply:
just stick a //nolint:gosec there

Reply:
> just stick a //nolint:gosec there

I'm getting 
```
api/rpc/client/client.go:31:42: directive `//nolint:gosec` is unused for linter "gosec" (nolintlint)
	authEnvKey = "CELESTIA_NODE_AUTH_TOKEN" //nolint:gosec
```